client/lib/invites/test/invites-create-validation.js

```diff
@@ -0,0 +1,45 @@
+require( 'lib/react-test-env-setup' )();
+
+const assert = require( 'chai' ).assert;
+
+/**
+ * Internal dependencies
+ */
+const Dispatcher = require( 'dispatcher' ),
+	constants = require( 'lib/invites/constants' );
+
+describe( 'List Invites Store', function() {
+	let InvitesCreateValidationStore;
+	const siteId = 123;
+	const actions = {
+		receiveValidaton: {
+			type: constants.action.RECEIVE_CREATE_INVITE_VALIDATION_SUCCESS,
+			siteId: siteId,
+			data: {
+				errors: [ 'asdfasdf' ],
```

Comment:
While this test works, I don't think it matches what we'll want from the API. On an error, we'll probably receive an array of objects, where each object contains the invitee as well as the error message. That currently looks like:

```
{
	"errors": {
		"test@gmail.com": {
			"errors": {
				"form-error-username-or-email": [
					"User already has a role on your site."
				]
			},
			"error_data": []
		}
	},
	"success": []
}
```

We could simplify errors a bit to be something like this perhaps:

```
errors: [
  {
    invitee: "test@gmail.com",
    error: "form-error-username-or-email",
    error_message: "User already has a role on your site."
  }
]
```